config/metadata/configMeta.yaml

```diff
@@ -1047,11 +1052,44 @@ groups:
           This list is ignored when Type is "redis". The format is a list of
           strings of the form "scheme://host:port".
 
+  - name: GooglePeerManagement
+    title: "Google Peer Management"
+    description: >
+      controls how the Refinery cluster communicates between peers when using
+      Google Pub/Sub. Only applies when `PeerManagement.Type` is "google".
+
+    fields:
+      - name: Topic
+        v1group: PeerManagement
```

Comment:
These two `v1` lines can come out here and below; they only apply to settings that existed in 1.x versions of Refinery.